docs/release_process.md

```diff
@@ -23,31 +18,11 @@ Specifically, for this repository, there are two long living branches:
 
 To cut a release, follow these steps:
 
-1. Create a new PR for the `develop` branch with the following contents
-   1. bump the version, while following the semantic release strategy
-   2. Review and update the changelog
-      1. Ensure the current entries are correct
-      2. Add new entries where needed
-      3. Add the release date
-      4. Create a new `Unreleased` section
-2. Once the PR is merged to the `develop` branch ensure the `Preview` workflow passes
-   1. Use the artifacts from the `Preview` workflow to test/preview the release
-   2. See the [Preview workflow section](#preview-workflow) for more detail
-3. Create another PR for the `main` branch with the following contents
-   1. Merge commits from the `develop` branch
-4. Once the PR is approved and merged, create a release tag of that commit to `main`
-   1. Use the same release version from the earlier step and ensure the commit is signed
-
-      ```sh
-      git tag --sign --message "Tag release v0.1.2" v0.1.2
-      git push --tags
-      ```
-
-   2. This will trigger the `Release` workflow, which requires approval to continue
-      1. See the [Release workflow section](#release-workflow) for more detail
-
-If anything fails, please go read the respective action's log output and configuration
-file to reverse engineer your way to a fix/soluton.
+1. Go to the [`Release` Workflow](https://github.com/phylum-dev/phylum-ci/actions/workflows/release.yml) for the repo
+2. Select the `Run workflow` dropdown
+   1. The workflow can only be run on the `main` branch
```

Comment:
Is there something enforcing this?

Reply:
Yes...in the `release.yml` workflow:

```
...
jobs:
  release:
    name: Build, Verify, Release, and Publish
    if: github.ref_name == 'main'
...
```

That `if` will ensure only the `main` branch is allowed to trigger the release job.